addons/l10n_fi/data/account_tax_report_line.xml

```diff
@@ -10,6 +10,14 @@
         <field name="sequence">0</field>
         <field name="report_id" ref="vat_report"/>
     </record>
+    <record id="tax_report_sales_25_5" model="account.tax.report.line">
+        <field name="name">25.5 %:n vero</field>
+        <field name="code">sale_25_5</field>
+        <field name="tag_name">fi_301</field>
```

Comment:
shouldn't the tag name be different from the report line below? :thinking:

Reply:
Indeed, i couldn't find its use here but it's used in later versions so i will pay attention to that during FW